addons/web/static/src/js/widgets/debug_manager.js

```diff
@@ -271,23 +271,25 @@ DebugManager.include({
         });
     },
     get_view_fields: function () {
-        var self = this;
-        var model = this._action.res_model;
-        this._rpc({
-                model: model,
-                method: 'fields_get',
-                kwargs: {
-                    attributes: ['string', 'searchable', 'required', 'readonly', 'type', 'store', 'sortable', 'relation', 'help']
-                },
-            })
-            .done(function (fields) {
-                new Dialog(self, {
-                    title: _.str.sprintf(_t("Fields of %s"), model),
-                    $content: $(QWeb.render('WebClient.DebugManager.Action.Fields', {
-                        fields: fields
-                    }))
-                }).open();
+        var model = this._action.res_model,
+            self = this;
+        self._rpc({
+            model: 'ir.model',
+            method: 'search',
+            args: [[['model', '=', model]]]
+        }).done(function (ids) {
+            self.do_action({
+                res_model: 'ir.model.fields',
+                name: _t('View Fields'),
+                views: [[false, 'list'], [false, 'form']],
+                domain: [['model_id', '=', model]],
+                type: 'ir.actions.act_window',
+                context: {
+                    'default_model_id': ids[0],
+                    'search_default_group_by_ttype': 1
```

Comment:
I am not convinced by that default, usually, I just want to check the existence of a field but without really knowing the type.

Reply:
Ok, got it, at the end it is not mandatory and it add only 1 click more, no big deal, I will take your word here also.

Reply:
Well not really against it, not really in favour, I won't mind if others think it is a good addition :-)

Reply:
no problem, that make sense what you say, for me it is ok, I have the search to group by in 2 more clicks, and that's enough for me.